src/pages/student/student.jsx

```diff
@@ -144,13 +151,17 @@ const StudentView = () => {
               6: '6th Grade',
             },
             ethnicityOptions: [
-              'white',
-              'black',
-              'asian',
-              'latinx',
-              'american indian or alaska native',
-              'non-specified',
+              { value: 'white', label: 'white' },
+              { value: 'black', label: 'black' },
+              { value: 'asian', label: 'asian' },
+              { value: 'latinx', label: 'latinx' },
```

Comment:
nitpick - can you capitalize ethnicity and gender values on the frontend. you will still need to pass in lowercase values to the backend since that's how we store it.